### ISSUE TITLE
Spotify not found. Using Windows store version

### ISSUE THREAD

**Issue by Gamershy** [Xeroday/Spotify-Ad-Blocker#269]
EZBlocker Version: 1.8.1.0
Spotify Version: 1.1.84.716

Simply unable to detect or hook into Spotify, doesn't mute or anything anymore.

**Comment by Astrosp** [comment]
use **EZBlocker Version: 1.7.6.0**
its work like charm :-)

**Comment by vagend** [comment]
Spotify not found.

Versions 1.8 and 1.76

**Comment by Gamershy** [comment]
Honestly, I gave up on EZBlocker. Switched to Spicetify.

**Comment by Astrosp** [comment]
> Honestly, I gave up on EZBlocker. Switched to Spicetify.

Did Spicetify block ads

**Comment by Gamershy** [comment]
Yep. It's a modded client for Spotify that allows for custom themes, features, and plugins. One of the plugins is an ad blocker that completely blocks all banner ads, audio ads, and video ads. Doesn't even have silence from muting the ads, they just don't exist. It's like having premium, without premium.

**Comment by gihanadnivak** [comment]
> Honestly, I gave up on EZBlocker. Switched to Spicetify.
what are the steps for muting ads ?

**Comment by mendelsphotography** [comment]
never heard of that is that open source? or probably not if it's a mod.
This was just muting so 1. Safe and open source 2. It's not a crack/Mod.... and don't see anything illegal with this vs mods.... that's what I loved about it. It's a shame the project died down tough.

**Comment by YagneshDave** [comment]
> Honestly, I gave up on EZBlocker. Switched to Spicetify.

its CLI ezblocker have graphic interface

**Comment by Kaeram** [comment]
Worked like a charm on win10, Spotify not found on win 11

**Comment by TitouanGisle** [comment]
Had the issue on windows 11. Managed to make it work by executing v1.7.6 on the desktop.
